app/client/mindjogg/components/Textbox/Textbox.md

```diff
@@ -0,0 +1,26 @@
+## Textbox
+The textbox is a component which provides an area for text input to be used in forms. It is possible to display an icon next to the
+text input area through props. 
+
+## Props
+| Prop         | Desc     | Default Value | Type | Required |
+|--------------|-----------|------------|--------|----------|
+| Name | The name this field will be accessed by in a form     | "myTextbox"        |  String              | Yes|
+| maxLength      | Maximum length (in characters) of accepted input  | 1000       |  Number    | No|
+| minLength      | Minimum length (in characters) of accepted input  | 1       |  Number    | No|
+| Placeholder      | Text to be shown before any input provided  | "Enter text here"       |  String    | No|
+| required      | Is input required in this field before the form can be submitted  | False       |  Boolean    | No|
+| iconURI      | A URI where an image to be used as an icon is located  | null       |  String    | No|

```

Comment:
@ja-chisholm don't we using Meterial Icon or vector Icon

Reply:
We do, but since we were having issues with Material this is temporary. Once we finalize an icon library it will be updated

Reply:
@ja-chisholm The problem we faced was because we downloaded the Meterial Icon directly but expo already downloaded the React Native vector icon which include Meterial Icon

Reply:
We should import like this:

``` import MaterialCommunityIcons from "react-native-vector-icons/MaterialCommunityIcons"; ```

Reply:
Sounds good, will update this PR and the password box

Reply:
@ja-chisholm  any update on the icon? you can also use "```react-native-vector-icons/Iconicons```" package alternatively.

Reply:
Icons are in now. The markdown file didn't get updated when I brought them in, but there is now documentation for using them. I used MaterialCommunityIcons